### ISSUE TITLE
Dropdown plugin classes in Vue3 app not working?

### ISSUE THREAD

**Issue by jimmiejackson** [htmlstreamofficial/preline#137]
I've got a Vue3 app that uses a Preline dropdown to build out a custom select, and while the dropdown does open, it doesn't appear like I'm able to use any of the plugin classes that should come along. 

I actually have the same issue as #42, and noticed that when attempting to add the class of `[--strategy:absolute]` to the menu, it's not actually being applied to the position property. After realizing that, I also noticed that the `hs-dropdown-open:rotate-180` class isn't working on an icon.

What might I be missing during setup?

_package.json_
```json
{
  "name": "test",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview",
    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
  },
  "dependencies": {
    "@heroicons/vue": "^2.0.18",
    "@vueuse/core": "^10.4.1",
    "preline": "^1.9.0",
    "vue": "^3.3.4",
    "vue-router": "^4.2.4"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^4.3.4",
    "autoprefixer": "^10.4.16",
    "eslint": "^8.49.0",
    "eslint-plugin-vue": "^9.17.0",
    "postcss": "^8.4.30",
    "tailwindcss": "^3.3.3",
    "vite": "^4.4.9"
  }
}
```

_tailwind.config.js_
```js
/** @type {import('tailwindcss').Config} */
import defaultTheme from 'tailwindcss/defaultTheme';
import prelinePlugin from 'preline/plugin';

export default {
  content: [
    'node_modules/preline/dist/*.js',
    './index.html',
    './src/**/*.{vue,js,ts,jsx,tsx}',
  ],
  plugins: [
    prelinePlugin,
  ],
};

```

_main.js_
```js
import { createApp } from 'vue';

import App from './App.vue';
import router from './router';

import('preline');

import '@/assets/css/main.css';

const app = createApp(App);

app.use(router);

app.mount('#app');
```

_Select.vue_
```vue
<template>
  <div class="flex flex-col w-full">
    <div class="hs-dropdown relative ">
      <button
        id="hs-dropdown-default"
        type="button"
        class="align-middle focus-visible:outline-primary font-medium gap-2 inline-flex justify-start items-center pl-4 pr-10 relative rounded text-base text-black transition-all w-full focus:outline-none focus:ring-none"
        :class="[
          { 'opacity-100 cursor-pointer bg-white hover:bg-gray-50 hs-dropdown-toggle': !disabled || !loading },
        ]">
        Actions
        <ChevronDownIcon
          class="absolute right-2 duration-300 ease-in-out font-bold h-5 hs-dropdown-open:rotate-180 stroke-2 text-primary transition-transform w-5"
          aria-hidden="true" />
      </button>
      <ul
        class="bg-white duration-[0.1ms] hidden hs-dropdown-menu hs-dropdown-open:opacity-100 min-w-[15rem] mt-2 opacity-0 p-2 rounded-lg shadow-md transition-[opacity,margin] w-72 z-10 [--strategy:absolute]"
        aria-labelledby="hs-dropdown-default">
       // ...<li /> options removed for brevity
      </ul>
    </div>
  </div>
</template>
```

**Comment by MangoMarcus** [comment]
> I also noticed that the hs-dropdown-open:rotate-180 class isn't working on an icon

I've been experiencing the same issue, looks like this was raised before in #82. I've suggested a workaround for now, here: https://github.com/htmlstreamofficial/preline/issues/82#issuecomment-1749494586

**Comment by jimmiejackson** [comment]
> I've been experiencing the same issue, looks like this was raised before in #82. I've suggested a workaround here: [#82 (comment)](https://github.com/htmlstreamofficial/preline/issues/82#issuecomment-1749494586)

Brilliant, thanks for that!

I tried using the same logic to work around the strategy class issue using `group-[.open]:!absolute` on the <ul>, though it causes some weird side effects like not displaying until I scroll down the page a bit.

**Comment by jahaganiev** [comment]
Hey folks, this issue has been addressed in v2.0 Preline. Thanks!
